Configuration/StandardSequences/python/EventInterpretation.py

```diff
@@ -1,3 +1,4 @@
 EventInterpretation = {
-    'top' : 'CommonTools.ParticleFlow.EITopPAG_cff'
+    'top' : 'CommonTools.ParticleFlow.EITopPAG_cff',
+    'MiniAODfromMiniAOD' : 'PhysicsTools.PatAlgos.slimming.MiniAODfromMiniAOD_cff'
```

Comment:
I suggested this as a hack.
Full implementation should deal with this as a separate step or a separate option in the configBuilder.
(plan to have it in 76X)